### DISCUSSION TITLE
"The warning has been cleared for p-link.io" 7 days ago but not cleared

### DISCUSSION THREAD

**Discussion by paracetamol951** [phantom/docs#270] [Help & Support] [answered]
### Summary

I wrote to blowfish, in order to ask for the deletion of the dangerous transaction, warning message in phantom.
On the 7th of august, i received "The warning has been cleared for [p-link.io](http://p-link.io/)"
I wrote again to blowfish, but they say they can nothing more, they dont know.
Yet, as i am writing this message, the warning is still there.
Is there an URL i can fetch in order to check the list ?
How long should it take ? Is it normal ?

### Example

https://p-link.io

### Steps to Reproduce

Click on the Pay button, that should open the transaction, a red message is shown : 
This app has been listed on blowfish list (automated) is is nost likely a scam (sorry i translated).
This message should have disappeared.

### Is there an existing discussion for this?

- [X] I have searched the existing discussions

**Comment by adamdelphantom** [comment, marked as answer]
Looking into this

**Comment by paracetamol951** [comment]
Hi,
It just disappeared a few minutes ago. Don't know if it is you, or if i just had to wait.
Thank you.
